src/Shared/Microsoft.DotNet.Scaffolding.Shared/T4Templating/CallContext.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.DotNet.Scaffolding.Shared.T4Templating
+{
+    internal sealed class CallContext
```

Comment:
Is this used anywhere?

Reply:
yes it is, in the generated .cs templates, I'll update the PR description and add a comment in this file.